README.md

```diff
@@ -35,8 +35,11 @@ applications that run on BEAM. Inspired by many of such lists over the GitHub.
 - [`metrics`](https://github.com/benoitc/erlang-metrics) - generic interface to to a different metrics systems in Erlang
 - [Prometheus](https://github.com/deadtrickster/prometheus.erl) - Prometheus integration for Erlang
 - [Statix](https://github.com/lexmag/statix) - fast and reliable Elixir client for StatsD-compatible servers with some DogStatsD extensions (namely tags)
-- [Telemetry.Poller](https://github.com/beam-telemetry/telemetry_poller) - periodically gather measurements and publish them as Telemetry events
 - [Telemetry](https://github.com/beam-telemetry/telemetry) - dynamic dispatcher for Erlang metrics and instrumentations
+- [Telemetry.Poller](https://github.com/beam-telemetry/telemetry_poller) - periodically gather measurements and publish them as Telemetry events
+- [Telemetry.Metrics](https://github.com/beam-telemetry/telemetry_metrics) - common interface for defining metrics based on :telemetry events.
+- [TelemetryMetricsStatsd](https://github.com/beam-telemetry/telemetry_metrics_statsd) - StatsD compatible Telemetry.Metrics reporter.
+- [TelemetryMetricsPrometheus](https://github.com/beam-telemetry/telemetry_metrics_prometheus) - Prometheus compatible Telemetry.Metrics reporter.
```

Comment:
I think I would add them as a sublist to the `Telemetry`.

Reply:
I like that. 👍